packages/astro/src/core/errors/errors-data.ts

```diff
@@ -168,9 +168,7 @@ export const NoMatchingRenderer = {
 
 ${
 	validRenderersCount > 0
-		? `There ${plural ? 'are' : 'is'} ${validRenderersCount} renderer${
-				plural ? 's' : ''
-			} configured in your \`astro.config.mjs\` file,
+		? `There ${plural ? 'are' : 'is'} ${validRenderersCount} renderer${plural ? 's' : ''} configured in your \`astro.config.mjs\` file,
```

Comment:
I like this change 😲 Does Biome preserve the single line even if the interpolated expression contains multiple identifiers? That has always bugged me from Prettier.

Reply:
Looks like a feature to me 😄